docs/parsers/MarkdownParser.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Markdown Parser
+author: williamabradley
+ms.date: 01/19/2018
+description: The Markdown Parser allows you to parse a Markdown String into a Markdown Document, and then Render it with a Markdown Renderer.
+keywords: uwp community toolkit, uwp toolkit, microsoft community toolkit, microsoft toolkit, markdown, markdown parsing, parser, markdown rendering
+---
+
+# Markdown Parser
+
+The [MarkdownDocument](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.parsers.markdown.markdowndocument) class allows you to parse a Markdown String into a Markdown Document, and then Render it with a Markdown Renderer.
+
+| Class | Purpose |
+| --- | --- |
+| **Microsoft.Toolkit.Parsers.Markdown.MarkdownDocument** | Represents a Markdown Document. |
+| **Microsoft.Toolkit.Parsers.Markdown.Render.MarkdownRendererBase** | A base renderer for Rendering Markdown into Controls. |
+
+## Syntax
+
+**C#**
+
+```csharp
+string md = "This is **Markdown**";
+MarkdownDocument Document = new MarkdownDocument();
+Document.Parse(md);
```

Comment:
Maybe add few lines here to demonstrate the output and how it can be used?

Reply:
Done.